src/components/ArticleList.js

```diff
@@ -1,24 +1,33 @@
-'use strict';
+"use strict";
 
-import ArticlePreview from './ArticlePreview';
-import React from 'react';
+import ArticlePreview from "./ArticlePreview";
+import React from "react";
 
-const ArticleList = props => {
-  if (!props.articles) {
-    return <div className="article-preview">Loading...</div>;
+class ArticleList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.articles = props.articles;
   }
 
-  if (props.articles.length === 0) {
-    return <div className="article-preview">No articles are here... yet.</div>;
-  }
+  render() {
+    if (!this.articles) {
```

Comment:
This would be better to be in a single the JSX section within the `return ( ... )` at the end of this render function

Reply:
would have to see how to condition different div elements within return.